src/Impro.js

```diff
@@ -1,13 +1,12 @@
-/*global JSON*/
-let _ = require('lodash');
-let mime = require('mime');
-let Pipeline = require('./Pipeline');
+var _ = require('lodash');
```

Comment:
Err, I can return there to being `let` - that said `lebab` passes are probably desirable.

Reply:
Yeah, `lebab` ftw. `eslint --fix` can also take care of many of those, if a good rule set is enabled.

Reply:
Oh yeah! My thought was if we land this as-is its be natural to then follow up separately with a prettier conversion and then run the latest eslint over the lot.

Reply:
Great! Yeah, let's do that.